package.json

```diff
@@ -26,10 +26,11 @@
     "postgenerate": "yarn lint:write",
     "preinstall": "yarn config set ignore-engines true",
     "postinstall": "git submodule update --init --recursive && patch-package",
-    "build": "is-ci && yarn build:ci || yarn build:dev",
-    "build:base": "turbo run build --filter=!embedded-accounts-quickstart --filter=!ui-demo",
-    "build:dev": "yarn build:base --filter=!docs",
-    "build:ci": "yarn build:base && yarn lint:write",
+    "build": "yarn build:libs",
```

Comment:
this is for backwards compat, I guess I could just merge this and `build:libs`, but I wanted the separation just to be explicit in the CI script